src/app/psat/help-panel/pump-operations-help/pump-operations-help.component.ts

```diff
@@ -9,9 +9,10 @@ import { Subscription } from 'rxjs';
 })
 export class PumpOperationsHelpComponent implements OnInit {
  
+  @Input()
```

Comment:
since the value of `currentField` is set from the subscription. Do not pass the value as an input as well